roles/arteria-checksum-ws/tasks/main.yml

```diff
@@ -5,24 +5,18 @@
 # Production 
 - set_fact: 
     arteria_checksum_monitored_path: "{{ arteria_checksum_monitored_path }}"
-    arteria_checksum_env_root: "{{ arteria_checksum_env_root }}/production/"
-    arteria_checksum_src_path: "{{ arteria_checksum_src_path }}/production/"
-    arteria_checksum_config_root: "{{ arteria_checksum_config_root }}/production/"
     arteria_checksum_md5sum_log_dir: "{{ arteria_checksum_md5sum_log_dir }}/production"
     arteria_checksum_service_log: "{{ arteria_checksum_service_log}}/production"
     arteria_checksum_port: "{{ arteria_checksum_port_prod }}"
-  when: arteria_checksum_environment == "production"
+  when: deployment_environment == "production"
 
 # Staging
 - set_fact:
     arteria_checksum_monitored_path: "{{ arteria_checksum_monitored_path }}/staging/"
-    arteria_checksum_env_root: "{{ arteria_checksum_env_root }}/staging/"
-    arteria_checksum_src_path: "{{ arteria_checksum_src_path }}/staging/"
-    arteria_checksum_config_root: "{{ arteria_checksum_config_root }}/staging/"
     arteria_checksum_md5sum_log_dir: "{{ arteria_checksum_md5sum_log_dir }}/staging"
     arteria_checksum_service_log: "{{ arteria_checksum_service_log}}/staging"
     arteria_checksum_port: "{{ arteria_checksum_port_stage }}"
-  when: arteria_checksum_environment == "staging"
+  when: deployment_environment in [ "staging", "devel" ]
```

Comment:
Yes. `devel` is the value of `deployment_environment` if you're running `ansible-playbook` on irma3 without specifying the `-e` flag to override it. I.e. the value that gets set when you're inside `devel/{isak,johanhe}/` and testing that the playbook works.